### DISCUSSION TITLE
Prompt label promotion via api/sdk

### DISCUSSION THREAD

**Discussion by mrdrprofuroboros** [langfuse/langfuse#3436] [Ideas]
### Describe the feature or potential improvement

I have a bunch of prompts for my agent and I use version labels to mark the environment they are used in. And once I've done modifying prompts in 'development' I want to promote them all to 'staging' or 'production' in bulk. So far I can either go to the UI and click labels manually or create new versions with appropriate tag.

That would be perfect if you could add update_labels or some similar API in PromptsClient.
The pattern of usage is "set label X on latest version of the prompt NAME". Or more general "set label X on the prompt NAME's version where the label Y is" - that way we'd be able to promote 'production' to 'staging' even if 'latest'/'development' is way ahead

**Comment by marcklingen** [comment]
thanks for sharing! can you add your upvote to this thread to stay in the loop? https://github.com/orgs/langfuse/discussions/2810

closing this as a duplicate
